versions.props

```diff
@@ -1,6 +1,6 @@
 ch.qos.logback:* = 1.1.3
 com.codahale.metrics:metrics-core = 3.0.2
-com.fasterxml.jackson.*:* = 2.6.7
+com.fasterxml.jackson.*:* = 2.8.6
```

Comment:
See above comment, but we need to find a way for `timelock-server` only to use 2.8.6 and all other modules to use 2.6.7 due to downstream version conflicts.